js/exporter/jspdf/pdf_grid.js

```diff
@@ -48,17 +48,54 @@ export class PdfGrid {
         let currentTableIndex = 0;
         let currentTableCells = [];
         for(let cellIndex = 0; cellIndex < cells.length; cellIndex++) {
+            const currentCell = cells[cellIndex];
             const isNewTableColumn = this._splitByColumns.filter((splitByColumn) => splitByColumn.columnIndex === cellIndex)[0];
             if(isNewTableColumn) {
                 this._currentHorizontalTables[currentTableIndex].addRow(currentTableCells, rowHeight);
+                if(currentCell.colSpan > 0 && currentCell.text === '') {
+                    const notEmptyColSpanCells = currentTableCells.filter(cell => cell.colSpan > 0 && cell.text.length > 0);
+                    if(notEmptyColSpanCells.length > 0) {
+                        currentCell.text = notEmptyColSpanCells[notEmptyColSpanCells.length - 1].text;
+                    }
+                }
                 currentTableIndex++;
                 currentTableCells = [];
             }
-            currentTableCells.push(cells[cellIndex]);
+            currentTableCells.push(currentCell);
         }
         this._currentHorizontalTables[currentTableIndex].addRow(currentTableCells, rowHeight);
     }
 
+    mergeCellsBySpanAttributes() {
+        this._tables.forEach((table) => {
+            for(let rowIndex = 0; rowIndex < table.rows.length; rowIndex++) {
+                for(let cellIndex = 0; cellIndex < table.rows[rowIndex].length; cellIndex++) {
+                    const cell = table.rows[rowIndex][cellIndex];
+                    if(!cell.skip) {
+                        if(isDefined(cell.rowSpan)) {
+                            for(let i = 1; i <= cell.rowSpan; i++) {
+                                const mergedCell = table.rows[rowIndex + i][cellIndex];
+                                if(isDefined(mergedCell)) {
+                                    cell._rect.h += mergedCell._rect.h;
+                                    mergedCell.skip = true;
+                                }
+                            }
+                        }
+                        if(isDefined(cell.colSpan)) {
+                            for(let i = 1; i <= cell.colSpan; i++) {
+                                const mergedCell = table.rows[rowIndex][cellIndex + i];
+                                if(isDefined(mergedCell) && isDefined(mergedCell.colSpan)) {
```

Comment:
There should not be "isDefined(mergedCell.colSpan)" condition.
At this point it means "nested merged region".
There should not be such condition.